components/Registration/Registration.tsx

```diff
@@ -11,6 +11,23 @@ import { registrationSchemas } from "./validation";
 import NavigationButton from "../Form/NavigationButton/NavigationButton";
 import { Formik, Form, FormikHelpers } from "formik";
 
+import PERSONAL_INFO from "@/public/registration/backgrounds/personal_info.jpg";
+import EDUCATION from "@/public/registration/backgrounds/education.jpg";
+import HACK_SPECIFIC from "@/public/registration/backgrounds/hack_specific.jpg";
+import TRANSPORTATION from "@/public/registration/backgrounds/transportation.jpg";
+import REVIEW_INFO from "@/public/registration/backgrounds/review_info.jpg";
+
+import PERSONAL_INFO_MOBILE from "@/public/registration/mobile_backgrounds/personal_info.jpg";
+import EDUCATION_MOBILE from "@/public/registration/mobile_backgrounds/education.jpg";
+import HACK_SPECIFIC_MOBILE from "@/public/registration/mobile_backgrounds/hack_specific.jpg";
+import TRANSPORTATION_MOBILE from "@/public/registration/mobile_backgrounds/transportation.jpg";
+import REVIEW_INFO_MOBILE from "@/public/registration/mobile_backgrounds/review_info.jpg";
+
+import ARTEMIS from "@/public/registration/characters/artemis.png";
+import APOLLO from "@/public/registration/characters/apollo.png";
+import NONE from "@/public/registration/characters/none.png";
```

Comment:
Can you save all of these images (backgrounds and characters) as SVGs from the Figma instead of jpg and png. This should make the backgrounds load faster

Reply:
I converted all of the characters and backgrounds (mobile and desktop) to SVG 👍